include/inria_wbc/behaviors/franka_circular_cartesian.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef IWBC_SWITCH_FOOT
+#define IWBC_SWITCH_FOOT
+#include <chrono>
+#include <iostream>
+#include <signal.h>
+
+#include <inria_wbc/behaviors/behavior.hpp>
+#include <inria_wbc/controllers/pos_tracker.hpp>
+#include <inria_wbc/estimators/cop.hpp>
+#include <inria_wbc/utils/trajectory_handler.hpp>
+
+namespace inria_wbc {
+    namespace behaviors {
+
+        class CircCartTraj : public Behavior {
+        public:
+            CircCartTraj(const controller_ptr_t& controller);
```

Comment:
As a general rule, we should have the same name for files & for classes. This class should have been FrankaCircularCartesian. However, I think we should try to not make these simple class robot-specific (same for the "arm" btw, which is simply a up-down motion that should be able to do with the Franka).

This will save code and make it easier to test on new robots (no need to develop new behaviors for the Tiago, for instance).

Reply:
changed name of the behavior class and also of the yaml file to : "circular_cartesian_trajectory "